source/howto/CreateAHeatStack.rst

```diff
@@ -1,11 +1,302 @@
-==============
+===================
 Create a Heat stack
-==============
+===================
 
-Documentations coming soon
+Heat is an orchestration component of OpenStack which allows templates to be used to define resources and to specify the relationships between resources in a stack.
 
+To check Heat is available we can run the command:
 
-######
-References
+.. code-block:: bash
+
+  openstack stack list
+
+This command returns the list of stacks in the user's project. If no stacks have been created, this command returns an empty line.
+
+To create a stack, we first need to write a template which will be used to launch the stack.
+
+##########
+Templates
+##########
+
+Templates are used to define the resources which should be created as part of a stack and the relationships between resources.
+With the integration of other OpenStack components, Heat Templates allow the create of most OpenStack resource types.
+This includes infrastructure resources such as instances, databases, security groups, users, etc.
+The Heat Orchestration Template (HOT) is native to OpenStack for creating stacks. These templates are written in YAML.
+
+
+Heat Orchestration Templates
+############################
+
+The structure of a Heat Orchestration Template is given as follows:
+
+.. code-block:: yaml
+
+  heat_template_version: 2018-08-31 #OpenStack Version we want to use.
+                                    #Here, we want to use template for the Rocky release onwards
+
+  description: #description of the template
+
+  parameter_groups: #declares the parameter group and order.
+    #This is not a compulsory section, however it is useful for grouping
+    #parameters together when building more complex templates.
+
+  parameters: #declares the parameters for resources
+
+  resources: #declares the template resources
+  # e.g. alarms, floating IPs, instances etc.
+
+  outputs: #declares the output of the stack
+
+  conditions: #declares any conditions on the stack
+
+heat_template_version
+#####################
+
+This tells heat which format and features will be supported when creating the stack.
+
+For example:
+
+.. code-block:: yaml
+
+  heat_template_version: 2018-03-02
+
+Indicates that the Heat Template contains features which have been added and/or removed up to the Queens release of OpenStack.
+
+The template version:
+
+.. code-block:: yaml
+
+  heat_template_version: 2018-08-31
+
+Indicates that the Heat Template contains features which have been added and/or removed up to the Rocky release of OpenStack.
+For templates being used in the Train release of OpenStack, this template version is used.
+
+A list of template versions can be found here: https://docs.openstack.org/heat/rocky/template_guide/hot_spec.html#hot-spec-intrinsic-functions
```

Comment:
This URL changes for each release, as we are now on the train release, could you link to the train version of this page?